### PR TITLE
Add install scripts

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add install scripts.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
None

### CODE REVIEW

File: docs/installation/installation.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ title: Installation Overview
 ### Karmada kubectl plugin
```
Comment (RainbowMango) [inline docs/installation/installation.md:7]: Here both `kubectl-karmada` and `karmadactl` are present in `Prerequistites`, do you mean people need to install them both?

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/install-cli-tools.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+---
+title: Installation of Cli Tools
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/installation/install-cli-tools.md:2]:
```suggestion
title: Installation of CLI Tools
```

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/install-cli-tools.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+## Install Karmadactl
+
+You can install `karmadactl` in any of the following ways:
+
+- One-click installation.
+- Build from source code.
+
+### One-click installation
+
+Karmada provides `karmadactl` plug-in download service since v1.2.0. You can choose proper plug-in version which fits your operator system form [karmada release](https://github.com/karmada-io/karmada/releases).
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/installation/install-cli-tools.md:13]: `karmadactl` is NOT PLUGIN.

File: docs/installation/install-cli-tools.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+## Install Karmadactl
+
+You can install `karmadactl` in any of the following ways:
+
+- One-click installation.
+- Build from source code.
+
+### One-click installation
+
+Karmada provides `karmadactl` plug-in download service since v1.2.0. You can choose proper plug-in version which fits your operator system form [karmada release](https://github.com/karmada-io/karmada/releases).
```
Comment (RainbowMango) [inline docs/installation/install-cli-tools.md:13]: not plug-in

File: docs/installation/installation.md (hunk 1 above)
Comment (Poor12) [reply docs/installation/installation.md:7]: Not yet. I think people install one of them is ok. Add it to the note.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**